src/Illuminate/Contracts/Pipeline/Piper.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Illuminate\Contracts\Pipeline;
+
+interface Piper
+{
+    /**
+     * Set the pipes through which commands should be piped before dispatching.
```

Comment:
Think you'll need to remove the "commands" and "dispatching" naming here since this will also apply for other implementations.

Reply:
`Set the pipes through which calls should be piped.`? Kind of repetitive but can't think of anything better.